cmd/controller/sparsefilegenerator.go

```diff
@@ -45,6 +45,9 @@ On Shutdown, the status of the sparse file Disk CR will be marked as Unknown.
 */
 
 const (
+	/*ENV_SPARSE_FILE_DIR - if it specified as a environment variable,
```

Comment:
mention that it will create Sparse file inside `ENV_SPARSE_FILE_DIR` dir

Reply:
Thank you for your review. It is fixed